scripts/migrations/018_remove_unwanted_field_from_clearing_request.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2020. Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# This is a manual database migration script. It is assumed that a
+# dedicated framework for automatic migration will be written in the
+# future. When that happens, this script should be refactored to conform
+# to the framework's prerequisites to be run by the framework. For
+# example, server address and db name should be parameterized, the code
+# reorganized into a single class or function, etc.
+#
+# This script updates releases with empty clearingState to default value - NEW_CLEARING.
+# -----------------------------------------------------------------------------
+
+import time
+import couchdb
+import json
+from webbrowser import get
+
+# ---------------------------------------
+# constants
+# ---------------------------------------
+
+DRY_RUN = False
+
+COUCHSERVER = "http://localhost:5984/"
+DBNAME = 'sw360db'
+
+couch = couchdb.Server(COUCHSERVER)
+db = couch[DBNAME]
+
+
+# set default values for clearingState field of Release
+fieldValue = "clearingTeamComment"
+
+
+# ----------------------------------------
+# queries
+# ----------------------------------------
+
+# get all the clearing requests
+cr_all_query = '''function(doc){
+    if (doc.type == "clearingRequest"){
+        emit(doc._id, doc)
+    }
+}'''

```

Comment:
Instead of fetching all CRs and then filtering in a for loop.
It would be good to fetch all Docs containing the field
```
'''function(doc){
    if (doc.type == "clearingRequest" && doc.'''+ fieldValue +''' !=null){
        emit(doc._id, doc)
    }
}'''
```

Reply:
done.